test/driver.spec.js

```diff
@@ -98,7 +98,33 @@ describe('did:key method driver', () => {
     });
   });
 
-  describe('computeId', async () => {
+  describe('publicMethodFor', () => {
```

Comment:
I'm curious to know what happens if I ask for a purpose that is not available.  There's no documentation or tests to indicate if it throws or what.

Reply:
Ah, good call.

Reply:
Added docs & test.